src/utils/themeStyles/index.ts

```diff
@@ -0,0 +1,31 @@
+import { ThemeColors, ThemeType } from '../../constants';
+
+export const primaryButtonFontColor = (theme: ThemeType) =>
```

Comment:
This is great! I wonder if we should add a comment at the top of the file that explains that any ternary that is checking `theme.__type` should probably be moved into a util function in this file? (With the one exception being the Icon component, as you left it).

This isn't not a blocking comment--thank you for this cleanup 🙂